guides/v2.2/architecture/archi_perspectives/present_layer.md

```diff
@@ -29,14 +28,6 @@ Each [theme](https://glossary.magento.com/theme) resides in a unique directory a
 
 For an extensive introduction to theme elements and an overview of how to extend and override the default Magento themes, see the [Frontend Developer Guide][].
 
-## GraphQL
-
-GraphQL is a data query language developed internally by Facebook in 2012 before being publicly released in 2015. Magento implements GraphQL to provide an alternative to REST and SOAP web APIs for frontend development.
-
-GraphQL allows you to define the structure of the data that you need, and the server returns only the data you request. Each GraphQL-capable module contains a declarative schema that defines the syntax for queries that the module supports, as well as the attributes that can be returned. If you run a REST call such as GET /V1/products/:sku on a simple product, the system might fetch more than 100 lines of data. If all you need is the current price, the call has returned significantly more information than you need. With GraphQL, a query against the same SKU could return just the price.
-
-More information can be found in the [GraphQL Developer Guide][]
-
 ## Progressive Web Apps
```

Comment:
@jcalcaben Should PWA be mentioned in a 2.2 document?

Reply:
no, PWA Studio is 2.3+

Reply:
This is out of scope in this pull request. But I removed the sections about PWA, there were two of them with the same heading. Also, I removed the duplicated section in 2.3.